pilot/pkg/model/push_context.go

```diff
@@ -1627,10 +1627,12 @@ func (ps *PushContext) initEnvoyFilters(env *Environment) error {
 
 // EnvoyFilters return the merged EnvoyFilterWrapper of a proxy
 func (ps *PushContext) EnvoyFilters(proxy *Proxy) *EnvoyFilterWrapper {
+	fmt.Printf("MATCHING ENVOY FILTERS: %#v\n", proxy)
```

Comment:
cleanup

Reply:
Yeah, sorry. I'm debugging why envoy filters are not passing through pilot.